### ISSUE TITLE
Drinking Joe then suiciding will make the effect permanent

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#20759]
(WEB REPORT BY: cowboybuttsdrivemenuts REMOTE: 172.93.109.202:7777)
# Revision
6752a2d2f9e752c4d5ac6aa1744f2bbf6112b55a

# Description
Drinking Joe then committing suicide by gun makes the black and white color overlay permament after being cloned.
# Steps to Reproduce
1. Drink Joe
2. Suck on an egun
3. get cloned
4. no color vision
# What you Expected
To not have a black and white overlay when I come back to life
# What Actually Happened
Permanent color vision lost.

**Comment by DamianX** [comment]
Duplicate of #18491
blocked on pomf updating to 1454
